blog/2024-07-28.md

```diff
@@ -0,0 +1,56 @@
+---
+title: One place for all Smart Contracts Docs
+authors: [gagdiez]
+slug: sdks-unified
+tags: [updates]
+---
+
+*We have consolidated all our documentation in a single section, so you don't need to go searching around for it*
+
+<p><img src="/assets/images/contracts-landing-5a9c76a78e71b0e5f9a96033f1f23d23.png" /></p>
+
+<!-- truncate -->
+
+Smart contracts are small pieces of logic that can live on every NEAR account. To build a contract you use the NEAR SDKs, which comes in two flavours: Rust and JavaScript.
+
+Until today, we had multiple docs explaining how to build smart contracts:
+- `/sdk/rust` dedicated to explain how to use the Rust SDK
+- `/sdk/js` dedicated to explain how to use the JS SDK
+- [`/build/smart-contracts/what-is`](/build/smart-contracts/what-is) - that explains general concepts, and how to implement them using both SDKs
+
+Today, this is over, as all the information on how to build a smart contracts is located in a single area: [`/build/smart-contracts/what-is`](/build/smart-contracts/what-is).
+
+Meanwhile, we have transformed the [SDK page](/tools/sdk) to be a simple landing page with links to:
+- The [Rust SDK reference docs](https://docs.rs/near-sdk/latest/near_sdk/)
+- The [JS SDK reference docs](https://near.github.io/near-api-js/)
+- The [Smart Contract Section](/build/smart-contracts/what-is)
+
+## Why did we have 3 sections explaining the same?
+
+The reason we had 3 different sections was that, historically, the engineers of each SDK were working on their own docs in isolation. To help mitigate this, we created a section on NEAR docs, meant to consolidate all the external documentation.
+
+One day, the individual SDK pages were deleted - if I remember correctly, it was because we wanted to have less domains - and we had to migrate everything on a rush to here.
+
+This left us in a very weird situation: we already had a section explaining how to build a smart contract... and now we had 3.
+
+## A single source of truth
+
+Luckily, this is now fixed! We have finally conquered the original dream of having a single section for [Smart Contracts](/build/smart-contracts/what-is), with all the information consolidated in there. 
+
+Now, we can focus on maintaining a single section, thus making it easier to keep it updated and relevant.
+
+## What's next?
+
+We are currently undergoing a process of **consolidating** all the documentation. This means that we are looking at all the sections that have overlapping information, and trying to merge them into a single place.
+
+This will not only improve the quality of our docs, but also make it easier for you to find the information you need. In fact, improving search is one of the main motors of this change, since we noticed that our search tool ([algolia](https://www.algolia.com/)) gets confused when the same concept is spread all over the place.
```

Comment:
```suggestion
This will not only improve the quality of our docs, but also make it easier for you to find the information you need. In fact, improving search is one of the main motors of this change, since we noticed that our search tool ([Algolia](https://www.algolia.com/)) gets confused when the same concept is spread all over the place.
```